src/__tests__/rules.test.ts

```diff
@@ -76,4 +77,138 @@ describe('rules', () => {
       expect(() => require(`../rules/${ruleName}`)).not.toThrow();
     });
   });
+
+  describe('rule documentation files have the correct content', () => {
+    enum MESSAGE_TYPE {
+      CONFIGS = 1,
+      DEPRECATED = 2,
+      FIXABLE = 3,
+      HAS_SUGGESTIONS = 4,
+    }
+    const MESSAGES = {
+      [MESSAGE_TYPE.CONFIGS]:
+        '💼 This rule is enabled in the following [configs](https://github.com/jest-community/eslint-plugin-jest#shareable-configurations):',
+      [MESSAGE_TYPE.DEPRECATED]: '❌ This rule is deprecated.',
+      [MESSAGE_TYPE.FIXABLE]:
+        '🔧 This rule is automatically fixable using the `--fix` [flag](https://eslint.org/docs/latest/user-guide/command-line-interface#--fix) on the command line.',
+      [MESSAGE_TYPE.HAS_SUGGESTIONS]:
+        '💡 This rule is manually fixable by editor [suggestions](https://eslint.org/docs/developer-guide/working-with-rules#providing-suggestions).',
+    };
+
+    function getConfigsForRule(ruleName: keyof typeof plugin.rules) {
+      const { configs } = plugin;
+      const configNames: Array<keyof typeof configs> = [];
+      let configName: keyof typeof configs;
+
+      for (configName in configs) {
+        const config = configs[configName];
+        // eslint-disable-next-line @typescript-eslint/ban-ts-comment -- we don't have a static type for rule names
+        // @ts-ignore
+        const value = config.rules[`jest/${ruleName}`];
+        const isEnabled = [2, 'error'].includes(value);
+
+        if (isEnabled) {
+          configNames.push(configName);
+        }
+      }
+
+      return configNames.sort();
+    }
+
+    function configNamesToList(configNames: string[]) {
+      return `\`${configNames.join('`, `')}\``;
+    }
+
+    ruleNames.forEach(ruleName => {
+      it(ruleName, () => {
```

Comment:
should use `it.each`: https://jestjs.io/docs/api#testeachtablename-fn-timeout

Reply:
Neat, updated to use that.